build-system/tasks/extension-helpers.js

```diff
@@ -770,7 +795,7 @@ async function generateBentoEntryPointSource(name, options) {
       );
     }
 
-    defineElement();
+    ${toExport ? '' : 'defineElement();'}
```

Comment:
nit: slightly nicer to keep the condition in one place.
```suggestion
    ${toExport ? 'export {defineElement};' : 'defineElement();'}
```